agasc/tests/test_agasc_2.py

```diff
@@ -581,3 +581,41 @@ def test_get_supplement_table_obs():
 def test_get_supplement_table_obs_dict():
     obs = agasc.get_supplement_table("obs", as_dict=True)
     assert isinstance(obs, dict)
+
+
+def test_write(tmp_path):
+    from pathlib import Path
+
+    import tables
+
+    from agasc import write_agasc
+
+    with tables.open_file(
+        Path(os.environ["SKA"]) / "data" / "agasc" / "agasc_1p7.h5"
```

Comment:
This is now failing on HEAD because the file in /proj/sot/ska/data/agasc is agasc1p7.h5 (no _).  I think.

Reply:
true

Reply:
I happen to have a link with the `_`. I don't know why.

Reply:
Yeah, I couldn't remember the filename history to know if we wanted to make the link you have or update the test to match what we've got in /proj/sot/ska.